help_to_heat/frontdoor/views.py

```diff
@@ -617,6 +630,19 @@ class PropertyTypeView(PageView):
     def get_context(self, *args, **kwargs):
         return {"property_type_options": schemas.property_type_options_map}
 
+    def get_prev_next_urls(self, session_id, page_name):
+        session_data = interface.api.session.get_session(session_id)
+        own_property = session_data.get("own_property")
+        country = session_data.get("country")
+        epc_rating = session_data.get("epc_rating", "Not found")
+
+        if own_property == "No, I am a social housing tenant" and (country == "Scotland" or epc_rating == "Not found"):
```

Comment:
Sanity-checking both changes as the ticket is ambiguous I think: is this definitely an `and`, not an `or`?

Reply:
![image](https://github.com/UKGovernmentBEIS/help-to-heat-GBIS/assets/152421975/f639b08e-de62-4eb9-8cd4-c994e4108113)
I haven't checked personally, but I believe it's "and" going by the backlink flow chart from the investigation.